docs/proposals/ldap-group-sync.md

```diff
@@ -0,0 +1,92 @@
+# Allowing Groups to be Based on and Synced With External LDAP Records
```

Comment:
@deads2k If we are just requiring users to supply opaque filter strings for LDAP requests like is done for @liggitt's implementation of the login, why do we need to specifically support any one schema?